app/src/main/java/org/sopt/sample/week1/login/LoginActivity.kt

```diff
@@ -0,0 +1,69 @@
+package org.sopt.sample.week1.login
+
+import android.content.Intent
+import android.os.Bundle
+import android.widget.Toast
+import androidx.appcompat.app.AppCompatActivity
+import org.sopt.sample.databinding.ActivityLoginBinding
+import org.sopt.sample.week1.common.Constants
+import org.sopt.sample.week1.common.MBTI
+import org.sopt.sample.week1.mypage.MyPageActivity
+import org.sopt.sample.week1.signup.SignUpActivity
+
+class LoginActivity : AppCompatActivity() {
+    private lateinit var binding: ActivityLoginBinding
+    private lateinit var insertedId: String
+    private lateinit var insertedPassword: String
+    private lateinit var insertedMbti: MBTI
+
+    override fun onCreate(savedInstanceState: Bundle?) {
+        super.onCreate(savedInstanceState)
+
+        binding = ActivityLoginBinding.inflate(layoutInflater)
+        setContentView(binding.root)
+
+        // intent 에 값이 로그인 정보가 존재하는 경우
+        if (intent.hasExtra(Constants.ID.value) && intent.hasExtra(Constants.PASSWORD.value)) {
+            insertedId = intent.getStringExtra(Constants.ID.value)
+                ?: throw java.lang.IllegalArgumentException("id expected")
+            insertedPassword = intent.getStringExtra(Constants.PASSWORD.value)
+                ?: throw java.lang.IllegalArgumentException("password expected")
+
+            // for version 28+
+            insertedMbti = (intent.getSerializableExtra(Constants.MBTI.value)
+                ?: throw java.lang.IllegalArgumentException("mbti expected")) as MBTI
+        }
+
+
+        // click signUp
+        binding.btnSignUp.setOnClickListener {
+            val toSignUpView = Intent(this, SignUpActivity::class.java)
+            startActivity(toSignUpView)
+        }
+
+        // click login
+        binding.btnLogin.setOnClickListener {
+
+            // 회원가입된 정보가 로그인 페이지로 전달되지 않은 경우
+            if (!intent.hasExtra(Constants.ID.value) || !intent.hasExtra(Constants.PASSWORD.value)) {
+                Toast.makeText(this.applicationContext, "등록되지 않은 회원입니다.", Toast.LENGTH_SHORT).show()
+                return@setOnClickListener
+            }
+
+            if (binding.etIdInsertion.text.toString() == insertedId &&
+                binding.etPasswordInsertion.text.toString() == insertedPassword
+            ) {
+                // login success
+                Toast.makeText(this.applicationContext, "로그인에 성공하셨습니다.", Toast.LENGTH_SHORT).show()
+                val toMyPageView = Intent(this, MyPageActivity::class.java)
+                toMyPageView.putExtra(Constants.ID.value, insertedId)
+                toMyPageView.putExtra(Constants.MBTI.value, insertedMbti)
+                startActivity(toMyPageView)
```

Comment:
```suggestion
                startActivity(toMyPageView)
                finish()
```

MyPage뷰로 이동되어도 Login뷰는 그대로 남아있겠네요!
`finish()`를 이용해보면 좋을 것 같아용